com.cubrid.common.ui/src/com/cubrid/common/ui/common/dialog/JdbcOptionDialog.java

```diff
@@ -82,7 +82,6 @@ protected Control createDialogArea(Composite parent) {
 		container.loadPreference();
 
 		setTitle(Messages.titleJdbcOptionDialog);
-		setMessage(Messages.msgJdbcManagementDialog);
```

Comment:
I also agree as you that is wrong message.
But I think that removed message's function is additional information to user.
How about update that message as properly rather than remove?
For example like below.
`Config the advanced JDBC properties`